packages/rancher-logging/generated-changes/patch/Chart.yaml.patch

```diff
@@ -10,7 +10,7 @@
 +  catalog.cattle.io/namespace: cattle-logging-system
 +  catalog.cattle.io/permits-os: linux,windows
 +  catalog.cattle.io/provides-gvr: logging.banzaicloud.io.clusterflow/v1beta1
-+  catalog.cattle.io/rancher-version: <2.5.99-0
++  catalog.cattle.io/rancher-version: '>2.6.0-0'
```

Comment:
nit: `'> 2.6.0-0'`, not `'>2.6.0-0'`. Space is missing compared to previous logging chart